reporting/impl/src/main/java/org/jboss/windup/reporting/export/ExportCSVFileRuleProvider.java

```diff
@@ -66,63 +70,108 @@ public void perform(GraphRewrite event, EvaluationContext context, WindupConfigu
             final Map<String, CSVWriter> projectToFile = new HashMap<>();
             final Iterable<InlineHintModel> hints = hintService.findAll();
             final Iterable<ClassificationModel> classifications = classificationService.findAll();
+            List reportableEvents = new ArrayList<>();
+            reportableEvents.addAll((List<InlineHintModel>)hints);
+            reportableEvents.addAll((List<ClassificationModel>)classifications);
 
             //try{} in case something bad happens, we need to close files
             try
             {
-                for (InlineHintModel hint : hints)
+                reportableEvents.stream().sorted((o1,o2) ->
```

Comment:
i changed this method a little bit in https://gist.github.com/mrizzi/b30dea66197ff3db5ae486a12ff8a526

Reply:
np :wink: